src/components/Chat/MessageStage.tsx

```diff
@@ -17,6 +23,17 @@ interface StageTitleProps {
 }
 
 const StageTitle = ({ isOpened, stage }: StageTitleProps) => {
+  const [addon, setAddon] = useState<OpenAIEntityAddon | undefined>();
+  const addonsMap = useAppSelector(AddonsSelectors.selectAddonsMap);
+
+  const match = stage.name?.match(/^[^(]*/);
```

Comment:
this regex match everything except including `(`
even empty string
e.g.
match(`123(45`) -> `123`
match(`(12345`) -> ``

Reply:
We get a stage from core, whose name for the addon will look like this: addonName(question). If it finds a match with the addonMap, then we will show the icon. For now, only after receiving the part before "(" we will be able to show the icon.